### DISCUSSION TITLE
Are there any samples showing how to write log messages that consume properties using ExceptionDetails?

### DISCUSSION THREAD

**Discussion by flipdoubt** [RehanSaeed/Serilog.Exceptions#241] [Q&A]
I have Serilog.Exceptions properly configured and enriching, but I'm not clear on how to efficiently make use of the data. Are there any examples showing how others use these new properties? Do you add these properties to your `outputTemplate`? I hoped this would serialize the exception passed to `LogError(ex)` more readable, but it had no effect like that. A sample project in the repo would help.

**Comment by RehanSaeed** [comment]
Do you mean how to use the extra logged properties? Normally people log them as JSON to ElasticSearch, Seq which allows them to then search the logs, run very complex queries by property and create nice visualisations. All of that is out of scope of this project.

**Comment by flipdoubt** [comment]
Hi, Rehan. I just meant a sample outputTemplate showing a typical usage. If you suggest the typical usage is to not include these additional properties in the output template but to instead query them in some sort of search tool, then I guess that is the answer I'm looking for. Perhaps I missed it in the Readme, but that part of the "user story" is what eluded me. When my output template includes "{Exception}", I expected to see more but was confused when I didn't.
